shortlister/model.py

```diff
@@ -15,6 +15,7 @@ class Applicant:
     name: str
     cv: str #path to cv
     scores: Dict[Criterion,str]
+    notes: List[str]
```

Comment:
`notes` should be a single string variable. A list of notes will make it hard to write an edit function later. Instead of appending new notes to a list, append new notes to a string, using `+`. That means Applicant objects should start with an empty string, and every new note should be added (with a space) e.g.:

```
my_applicant.notes += f" {new_note}"
```

Reply:
I thought it was the opposite. Since strings are immutable, wouldn't making changes be easier if I replace strings with new strings in a list?

Reply:
When we add editing, we'll allow editing the entire text, not specific words or chunks. It'll be easier to edit a single string.

Reply:
I see!